docs/rancher/csi-driver.md

```diff
@@ -277,6 +277,121 @@ Now you can create a new StorageClass that you intend to use in your guest Kuber
 
+## RWX volumes support
+
+### Prerequisites
+
+:::note
+
+To use this feature, you need to ensure the host harvester cluster version is **v1.4.0** or later.
+
+:::
+
+Before creating RWX volumes on the guest cluster, you need to create a corresponding RWX StorageClass on the host cluster.
+
+1. Create a RWX StorageClass on the host
+  ```
+  kind: StorageClass
+  apiVersion: storage.k8s.io/v1
+  metadata:
+    name: longhorn-rwx
+  provisioner: driver.longhorn.io
+  allowVolumeExpansion: true
+  reclaimPolicy: Delete
+  volumeBindingMode: Immediate
+  parameters:
+    numberOfReplicas: "3"
+    staleReplicaTimeout: "2880"
+    fromBackup: ""
+    fsType: "ext4"
+    nfsOptions: "vers=4.2,noresvport,softerr,timeo=600,retrans=5"
+  ```
+
+  ![](/img/v1.4/rancher/create-rwx-sc-host-cluster-01.png)
+
+  ![](/img/v1.4/rancher/create-rwx-sc-host-cluster-02.png)
+
+  ![](/img/v1.4/rancher/create-rwx-sc-host-cluster-03.png)
+
+2. Ensure the RBAC settings are up-to-date
+
+  To support RWX feature, the harvester CSI driver needs to have the new RBAC settings. You can follow the steps below to check the RBAC settings.
+
+    ```
+    # kubectl get clusterrole harvesterhci.io:csi-driver -o yaml
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRole
+    metadata:
+    ...
+      name: harvesterhci.io:csi-driver
+    ...
+    rules:
+    - apiGroups:
+      - storage.k8s.io
+      resources:
+      - storageclasses
+      verbs:
+      - get
+      - list
+      - watch
+    - apiGroups:
+      - harvesterhci.io
+      resources:
+      - networkfilesystems
+      - networkfilesystems/status
+      verbs:
+      - '*'
+    - apiGroups:
+      - longhorn.io
+      resources:
+      - volumes
+      - volumes/status
+      verbs:
+      - get
+      - list
+    ```
+
+3. Ensure the networkfs-manager pod(s) is running
+
+  ```
+  # kubectl get pods -n harvester-system | grep networkfs-manager
+  harvester-networkfs-manager-2pxhm                       1/1     Running     4 (34m ago)    3h41m
+  harvester-networkfs-manager-8tst2                       1/1     Running     4 (37m ago)    3h41m
+  harvester-networkfs-manager-xvkgp                       1/1     Running     4 (37m ago)    3h41m
+  ```
+
+4. Ensure the Harvester Version is at least v0.1.20
+
+   If you use the older version of the Harvester CSI driver, upgrade it to v0.1.20 or later.
+
+  ![](/img/v1.4/rancher/harvester-csi-driver-version.png)
+
+### Deploying and Usage
+
+1. Create a new StorageClass on the downstream cluster to associate with the RWX StorageClass on the host cluster
+  
+  ![](/img/v1.4/rancher/new-sc-associated-with-rwx.png)
+
+2. Create RWX PVC with the new StorageClass
+
+  ![](/img/v1.4/rancher/create-rwx-pvc-01.png)
+
+  ![](/img/v1.4/rancher/create-rwx-pvc-02.png)
+
+3. Verify the RWX PVC is created successfully
```

Comment:
```suggestion
1. Verify that the RWX PVC was created successfully.
```

Reply:
How do users know that PVC creation was successful? Should they look for a specific state or status?

Reply:
The user usually checks with the GUI display on a downstream cluster, as shown in the following figure.

Reply:
You didn't answer my question so I'll rephrase it. Which field on the GUI indicates that PVC creation was successful? Is it the state/status **Bound**?

Reply:
Yeah, sorry.

Check the status is **Bound** on the GUI.

Reply:
Thanks!